### ISSUE TITLE
Alias cannot be inferred: "ModelA" has multiple relations with "ModelB"

### ISSUE THREAD

**Issue by Otoru** [RobinBuschmann/sequelize-typescript#671]
**Versions**
- sequelize: 5.17.0
- sequelize-typescript: 1.0.0
- typescript: 3.5.3

**I'm submitting a ...** 
- [x] bug report
- [ ] feature request

**Actual behavior:**
When performing a query on the table Users have an unexpected error.

**Expected behavior:**
Perform the query correctly.

**Steps to reproduce:**
Data my `Users` object, when executing `User.findAll command ({include: [models.Company]});` I have the error `Alias cannot be inferred:" User "has multiple relations with" Company "`

**Related code:**

```ts
import * as sequelize from 'sequelize-typescript';


@sequelize.Table({
    indexes: [{
        name: 'name',
        fields: ['firstName', 'lastName'],
        using: 'BTREE'
    },{
        name: 'status',
        fields: ['active'],
        using: 'BTREE'
    }]
})
export default class User extends sequelize.Model<User> {
    @sequelize.Unique
    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    username: string;

    @sequelize.Unique
    @sequelize.IsEmail
    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    email: string;

    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    firstName: string;

    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    lastName: string;

    @sequelize.Unique
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    document: string;

    @sequelize.AllowNull(false)
    @sequelize.Column(sequelize.DataType.DATE)
    birthday: Date;

    @sequelize.Default(false)
    @sequelize.Column(sequelize.DataType.BOOLEAN)
    active: boolean;

    @sequelize.BelongsToMany(() => Company, () => CompanyOwner)
    companies: Company[];

    @sequelize.BelongsToMany(() => Company, () => CompanyWorker)
    contractors: Company[];

    @sequelize.CreatedAt
    createdAt: Date;

    @sequelize.UpdatedAt
    updatedAt: Date;
}


@sequelize.Table({
    indexes: [{
        name: 'name',
        fields: ['name'],
        using: 'BTREE'
    }]
})
@sequelize.DefaultScope({
    include: [
        { as: 'owners', model: () => User },
        { as: 'contributors', model: () => User }
    ]
})
export default class Company extends sequelize.Model<Company> {

    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    name: string;

    @sequelize.Unique
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    document: string;

    @sequelize.IsEmail
    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    email: string;

    @sequelize.AllowNull(false)
    @sequelize.Length({max: 64})
    @sequelize.Column(sequelize.DataType.STRING(64))
    phone: string;

    @sequelize.BelongsToMany(() => User, () => CompanyOwner)
    owners: User[];

    @sequelize.BelongsToMany(() => User, () => CompanyWorker)
    contributors: User[];
}


@sequelize.Table
export default class CompanyWorker extends sequelize.Model<CompanyWorker> {

    @sequelize.AllowNull(false)
    @sequelize.ForeignKey(() => User)
    @sequelize.Column(sequelize.DataType.INTEGER)
    workerId: number;

    @sequelize.AllowNull(false)
    @sequelize.ForeignKey(() => Company)
    @sequelize.Column(sequelize.DataType.INTEGER)
    companyId: number;
}


@sequelize.Table
export default class CompanyOwner extends sequelize.Model<CompanyOwner> {

    @sequelize.AllowNull(false)
    @sequelize.ForeignKey(() => User)
    @sequelize.Column(sequelize.DataType.INTEGER)
    ownerId: number;

    @sequelize.AllowNull(false)
    @sequelize.ForeignKey(() => Company)
    @sequelize.Column(sequelize.DataType.INTEGER)
    companyId: number;
}

```

What did I do wrong?
